src/test/tests.ts

```diff
@@ -69,6 +68,23 @@ describe('RedisPubSub', function () {
     });
   });
 
+  it('cleans up correctly the memory when unsubscribing', function (done) {
+    pubSub.subscribe('Posts', () => null).then(subId => {
```

Comment:
Yeah so for the unsub twice to check what we want you must subscribe at least twice. 
Otherwise, the reason for it to throw an error could be that it cannot find the redis subscription.
This way we are making sure someone (Maybe it would change sometime) is cleaning the internal subscription map.